src/nupic/engine/RegionImplFactory.cpp

```diff
@@ -225,24 +220,10 @@ static RegionImpl * createPyNode(DynamicPythonLibrary * pyLib,
     std::string fullNodeType = std::string(package) + std::string(".") +
                                std::string(nodeType.c_str() + 3);
 
-    // Check if node exists and continue if not
-    std::string nodePath = Path::join(getPackageDir(pyLib->getRootDir(), package), 
-      std::string(nodeType.c_str() + 3) + std::string(".py"));
-
-      if (!Path::exists(nodePath))
-        continue;
-
     void * exception = nullptr;
     void * node = pyLib->createPyNode(fullNodeType, nodeParams, region, &exception);
     if (node)
       return static_cast<RegionImpl*>(node);
-
-    if (exception)
```

Comment:
If we are trying to load the region in every package, the ones that don't contain it will return exceptions.  We don't want to throw an exception for those cases.